packages/react-styles/src/css/components/Topology/topology-pipelines.css

```diff
@@ -0,0 +1,334 @@
+}
+
+.pf-topology-pipelines__pill.pf-m-selected .pf-topology-pipelines__pill--background {
+  fill: var(--pf-topology-pipelines__pill--m-selected--Background--Fill);
+}
+
+.pf-topology-pipelines__pill.pf-m-selected.pf-m-info .pf-topology-pipelines__pill--background {
+  fill: var(--pf-topology-pipelines__pill--m-info--Background--Fill);
+}
+
+.pf-topology-pipelines__pill.pf-m-selected.pf-m-success .pf-topology-pipelines__pill--background {
+  fill: var(--pf-topology-pipelines__pill--m-success--Background--Fill);
+}
+
+.pf-topology-pipelines__pill.pf-m-selected.pf-m-warning .pf-topology-pipelines__pill--background {
+  fill: var(--pf-topology-pipelines__pill--m-warning--Background--Fill);
+}
+
+.pf-topology-pipelines__pill.pf-m-selected.pf-m-danger .pf-topology-pipelines__pill--background {
+  fill: var(--pf-topology-pipelines__pill--m-danger--Background--Fill);
+}
+
+/* Node label */
+.pf-topology-pipelines__pill__text {
+  font-size: var(--pf-topology-pipelines__pill__text--FontSize);
+  font-family: var(--pf-topology-pipelines__pill__text--FontFamily);
+  fill: var(--pf-topology-pipelines__pill--Color);
+}
+
+.pf-topology-pipelines__pill.pf-m-selected .pf-topology-pipelines__pill__text {
+  fill: var(--pf-topology-pipelines__pill--m-selected__text-Color);
+}
+
+.pf-topology-pipelines__pill.pf-m-selected.pf-m-info .pf-topology-pipelines__pill__text {
+  fill: var(--pf-topology-pipelines__pill--m-selected--m-info__text-Color);
+}
+
+.pf-topology-pipelines__pill.pf-m-selected.pf-m-success .pf-topology-pipelines__pill__text {
+  fill: var(--pf-topology-pipelines__pill--m-selected--m-success__text-Color);
+}
+
+.pf-topology-pipelines__pill.pf-m-selected.pf-m-warning .pf-topology-pipelines__pill__text {
+  fill: var(--pf-topology-pipelines__pill--m-selected--m-warning__text-Color);
+}
+
+.pf-topology-pipelines__pill.pf-m-selected.pf-m-danger .pf-topology-pipelines__pill__text {
+  fill: var(--pf-topology-pipelines__pill--m-selected--m-danger__text-Color);
+}
+.pf-topology-pipelines__pill.pf-m-selected.pf-m-success .pf-topology-pipelines__pill__text {
+  fill: var(--pf-topology-pipelines__pill--m-selected--m-skipped__text-Color);
+}
+.pf-topology-pipelines__pill.pf-m-selected.pf-m-running .pf-topology-pipelines__pill__text {
+  fill: var(--pf-topology-pipelines__pill--m-selected--m-running__text-Color);
+}
+.pf-topology-pipelines__pill.pf-m-selected.pf-m-skipped .pf-topology-pipelines__pill__text {
+  fill: var(--pf-topology-pipelines__pill--m-selected--m-skipped__text-Color);
+}
+.pf-topology-pipelines__pill.pf-m-selected.pf-m-canceled .pf-topology-pipelines__pill__text {
+  fill: var(--pf-topology-pipelines__pill--m-selected--m-canceled__text-Color);
+}
+.pf-topology-pipelines__pill.pf-m-selected.pf-m-pending .pf-topology-pipelines__pill__text {
+  fill: var(--pf-topology-pipelines__pill--m-selected--m-running__text-Color);
+}
+
+.pf-topology-pipelines__pill--status.pf-m-selected {
```

Comment:
Here I think you want to add a rule for `.pf-topology-pipelines__pill--status` to have `color: var(--pf-topology-pipelines__pill--status--color)` and then in all the modifiers you can redefine that variable to be the correct status color variable.